### DISCUSSION TITLE
Seam blocker modifier

### DISCUSSION THREAD

**Discussion by LukaBronzovic** [OrcaSlicer/OrcaSlicer#1701] [Ideas]
Hi,

Seam blocker would be a great addition. 
I prefer using random seams as most of the outer walls get sanded and painted. The ones that are hardly accessible by any sanding power tools require a lot of manual fiddling with sand paper and files. Not having seams in those areas would help a lot.
Yes we can use the seam painter, but then we lose the randomness of the seams in areas where they are allowed.
Of course, the slicer should raise alert if it cannot place a seam due to user abusing the "No seam modifier" feature.

Kind regards,

Luka

**Comment by SoftFever** [comment]
that's a good idea

**Comment by LukaBronzovic** [comment]
Ah, well I just found the solution to my problem. I didn't know right click on the seam painter blocks the seam on the painted surface. I can get away with that, but again, the modifier would be better and would allow for all those modifications to be premade in the CAD and loaded in the slicer.

**Comment by mickdekkers** [comment]
This would also be very useful for areas of an object that benefit from tight tolerances, such as press fit magnet slots. Seam painting works pretty well, but a volumetric seam blocker feature would be a great time saver when the model has a lot of places that need seam blocking

**Comment by ouroborus** [comment]
I'd like to also see a seam enforcer modifier. This would let me use generated shapes to put seams in difficult-to-paint areas.

**Comment by imagesurgery** [comment]
I literally just modelled seam blocking geo based on my CAD, only to discover that of all the options, I cannot use them as seam blocking modifiers. Would have been nice, it took less time to offset a few faces and make the geo than it will take me to paint the inside corners of all my drawer rails.

+1
